docs/index.md

```diff
@@ -35,59 +35,42 @@ features:
 
 Welcome to all new visitors!
 
-This project is still very new and the team is still working hard on a number
-of areas of the whole project. That includes the website, package repository and
-GitHub organisation as well as the editor itself.
-
-The below is just some items that we are currently working on that might answer
-any questions you have at this early stage of the project.
-
 ### Downloads and Releases
 
 We are constantly working on the editor and maintain both a rolling release and regular release model for our binaries.
```

Comment:
```suggestion
We are constantly working on the editor and maintain both rolling and regular releases.
```

Reply:
Fantastic, I wasn't fond of the original wording but couldn't quite decide how to change it. This has it just right, I'd say.